File: Java/Day2/Task2.java
import java.util.Scanner;
public class Task2{
	public static void main(String[] args){
	Scanner input = new Scanner(System.in);

	System.out.println("Enter first score: ");
	int score1 = input.nextInt();
	System.out.println();

	System.out.println("Enter second score: ");
	int score2 = input.nextInt();
	System.out.println();

	System.out.println("Enter third score: ");
	int score3 = input.nextInt();
	System.out.println();

	double average = score1 + score2 + score3 / 3;
		if (average >= 90 && average <= 100){
			System.out.println("A");
		}
		else if (average >= 80 && average <= 89){
			System.out.println("B");
		}
		else if (average >= 70 && average <= 79){
			System.out.println("C");
		}
		else if (average >= 60 && average <= 69){
			System.out.println("D");
		}
		else {
			System.out.println("F");
		}
			
	
	}
}

File: Python/Day2/task_2.py
score_1 = int(input("Enter first score: "))
score_2 = int(input("Enter second score: "))
score_3 = int(input("Enter third score: "))

average = score_1 + score_2 + score_3 / 3
if average >= 90 and average <= 100:
	print("A")
elif average >= 80 and average < 90:
	print("B")
elif average >= 70 and average < 80:
	print("C")
elif average >= 60 and average < 70:
	print("D")
else:
	print("F")